app/views/password_resets/show.html.haml

```diff
@@ -0,0 +1,12 @@
+%section.reset_password.container
+  .row
+    .col-sm-6.col-sm-offset-1
+      = form_tag password_resets_path do 
+        %header
+          %h1 Reset Your Password
+        %fieldset.form-group
+          = label_tag :password, "New Password"
+          = text_field_tag :password, nil, class: 'form-control'
```

Comment:
Certainly use `password_field_tag` for passwords, otherwise other people could see the new password typed behind the shoulder etc.